tests/react-test-app/package.json

```diff
@@ -15,18 +16,24 @@
     "xmlbuilder": "^4.2.1"
   },
   "resolutions": {
-    "moment": "2.19.1"
+    "moment": "2.19.1",
+    "graceful-fs": "4.2.4"
   },
   "devDependencies": {
     "babel-preset-react-native": "4.0.0",
     "eslint": "^4.18.2",
     "fs-extra": "^4.0.3",
     "install-local": "^1.0.0",
     "invariant": "^2.2.2",
-    "metro-react-native-babel-preset": "^0.45.0"
+    "metro-react-native-babel-preset": "^0.45.0",
+    "cmake-js": "^6.1.0"
```

Comment:
Again, I believe we're doing something wrong if our users are forced to install this peer dependency.

Reply:
this is our test app. users are not forced to install it.